event-app-api/models/index.js

```diff
@@ -1,7 +1,7 @@
 import { User } from "./user.js";
 import { Event } from "./event.js";
 
-User.hasMany(Event, { foreignKey: "username" });
-Event.belongsTo(User);
+User.hasMany(Event, { foreignKey: "userId" });
+Event.belongsTo(User, { foreignKey: "userId" });
```

Comment:
[no action required] I don't know if specifying the foreign key for event belong to user is doing anything. It isn't there in the wiki https://sequelize.org/docs/v6/core-concepts/assocs/#options-1